Pipfile

```diff
@@ -15,7 +15,7 @@ netaddr = "*"
 # WSGI server
 gunicorn = "*"
 # Debian stretch version
-Django = "~=1.11.0"
+Django = "==2.0.13"
```

Comment:
The version doesn't fit the one in the commit message. ^^

Reply:
True. I must have mixed up minor version here. However since it is later on upgraded to 2.2.13 and tested against this it should be fine.